client/src/components/AddCard/AddCard.tsx

```diff
@@ -0,0 +1,63 @@
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { FormikHelpers } from 'formik';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { setDefaultPayment } from '../../helpers/APICalls/setDefaultPayment';
+import AddCardForm from './AddCardForm/AddCardForm';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function AddCard(): JSX.Element {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+  const { loggedInUser } = useAuth();
+
+  const handleSubmit = (
+    { paymentMethod }: { paymentMethod: string },
+    { setSubmitting }: FormikHelpers<{ paymentMethod: string }>,
+  ) => {
+    if (loggedInUser != undefined && loggedInUser) {
```

Comment:
Prefer to use strict equality operator `===`, `!==` if you care about other falsy values, as all falsy values will always be equal when compared using `==` rather than `===`. Or simply test for truthiness if that's only what you're interested in.
` if (loggedInUser)`

Reply:
👍🏻Done